tmt/utils.py

```diff
@@ -626,14 +628,39 @@ def _run(self,
         stdout_thread.start()
         if not join:
             stderr_thread.start()
+
+        # A bit of logging helpers for debugging duration behavior
+        start_timestamp = time.monotonic()
+
+        def _event_timestamp() -> str:
+            return f'{time.monotonic() - start_timestamp:.4}'
+
+        def log_event(msg: str) -> None:
+            self.debug('command event', f'{_event_timestamp()} {msg}', level=4)
```

Comment:
```suggestion
            self.debug('Command event', f'{_event_timestamp()} {msg}', level=4)
```

Reply:
To be consistent with the `Command returned '124'.` message just below.

Reply:
Fixed in 45e9bd5c654d8c31645fc2e630f2d4ebcfdbd28d.